android/engine/build.gradle

```diff
@@ -54,6 +81,28 @@ dependencies {
     implementation 'joda-time:joda-time:2.10.6'
     implementation 'androidx.paging:paging-runtime-ktx:3.0.1'
 
+    //CQL
+    implementation group: 'xerces', name: 'xercesImpl', version: '2.11.0'
+    implementation(group: "com.github.java-json-tools", name: "msg-simple", version: "1.2");
+    implementation(group: "com.github.java-json-tools", name: "jackson-coreutils", version: "2.0");
+    implementation 'androidx.appcompat:appcompat:1.1.0'
+    implementation 'com.google.android.material:material:1.1.0'
+    implementation 'androidx.constraintlayout:constraintlayout:1.1.3'
+    implementation 'com.fasterxml.jackson.core:jackson-core:2.12.2'
+    implementation 'com.fasterxml.jackson.core:jackson-databind:2.12.2'
+    implementation 'com.fasterxml.jackson.core:jackson-annotations:2.12.2'
+    implementation 'org.codehaus.woodstox:woodstox-core-asl:4.4.1'
+    implementation 'javax.xml.stream:stax-api:1.0-2'
+    implementation "ca.uhn.hapi.fhir:hapi-fhir-android:5.3.0"
+    implementation 'org.opencds.cqf.cql:engine:1.5.2-SNAPSHOT'
+    implementation 'org.opencds.cqf.cql:engine.fhir:1.5.2-SNAPSHOT'
+    implementation 'org.opencds.cqf.cql:evaluator:1.2.0'
+    implementation 'org.opencds.cqf.cql:evaluator.builder:1.2.0'
+    implementation 'org.opencds.cqf.cql:evaluator.dagger:1.2.0'
+    testImplementation 'junit:junit:4.+'
```

Comment:
We can remove the these testing dependencies. Fhircore for example already uses Junit 5

Reply:
@ndegwamartin addressed